tests/InvokerTest.php

```diff
@@ -73,6 +73,15 @@ public function cannot_invoke_magic_method()
         $this->invoker->call([new InvokerTestMagicMethodFixture, 'foo']);
     }
 
+    /**
+     * @test
+     */
+    public function cannot_invoke_static_magic_method()
+    {
+        $this->expectException(NotCallableException::class);
```

Comment:
I added a test, but **without** next line:
```php
$this->expectExceptionMessage('Invoker\Test\InvokerTestMagicMethodFixture::foo() is not a callable. A __call() or __callStatic() method exists but magic methods are not supported.');
```

I can add this line after #40 will be merged.

Reply:
👍  #40 merged